inc/templates/voting-system.html

```diff
@@ -7,7 +7,7 @@
 		{% else %}
 			<span>
 		{% endif %}
-				<a data-comment-id="{{ comment_id }}" class="vote-up" href="#">&#9650;</a>
+				<a data-comment-id="{{ comment_id }}" class="vote-up" href="#"><i class="material-icons">thumb_up</i></a>
```

Comment:
Is it just these 3 icons that have been replaced? I think it's better for implementors to use CSS to modify the icons if they need to. Loading the entire Material UI icon font seems like overkill for this.